### DISCUSSION TITLE
반성합니다..

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#583] [6. 반성문]
안녕하세요.. 또 반성문입니다...!

이번주는 글을 꼭 작성해보도록 하겠습니다~~ 인턴 시작하기 전까지 쓸 주제가 없었는데 곧 생기지 않을까 싶네용!!

**Comment by shkisme** [comment]
센디 주소 배치 해줘,,

**Reply by 02ggang9** [reply]
좋습니다~~ 잼있겠네유~~~~
